pkg/hope/node_management.go

```diff
@@ -86,7 +87,7 @@ func setupCommonNodeRequirements(log *logrus.Entry, node *Node) error {
 	return nil
 }
 
-func CreateClusterMaster(log *logrus.Entry, node *Node, podNetworkCidr string, force bool) error {
+func CreateClusterMaster(log *logrus.Entry, node *Node, podNetworkCidr string, loadBalancer *Node, loadBalancerHost string, masters *[]Node, force bool) error {
```

Comment:
Might be worth having multiple functions with how much the load balanced path brings in?

Reply:
Ended up making the branches more explicit. Don't think more is needed yet.